### ISSUE TITLE
Re-introduce chase effects on gate pass

### ISSUE THREAD

**Issue by HazardCreative** [RotorHazard/RotorHazard#150]
The LED animations were removed to prevent code execution from stopping during LED change effects. These can be re-introduced by using gevent's pseudo-threading to run the animations in parallel with other tasks.

**Comment by pulquero** [comment]
Given leds have limited interactions with the rest of the system, they can probably be multiprocessed and offloaded to another core.

**Comment by HazardCreative** [comment]
Certainly. I'll need to research that.

**Comment by ethomas997** [comment]
So what exactly is the effect we're looking for?  How would it fit in with signaling gate enter/exit status on the LEDs?

I'm planning on supporting LED control in the Arduino node code, with the server being able to send commands for LED colors and patterns, etc.  Now that we can plug in extra nodes via USB, this opens up some cool possibilities for multiple LED strips and panels.

**Comment by HazardCreative** [comment]
Delta5 had the "chase" animation play on gate crossing. I changed it since it caused a momentary pause in server responsiveness. I logged this issue with the loss of that visual effect, thinking we would just make the chase effect non-blocking and re-introduce it. 

The recent code is going in a better direction instead. Now, I think we should offer various selectable options for each timer event. (But, of course, these must continue to be non-blocking.) Let's just close this.
